Fix Data TCK

Pandrex247 opened the pull request:
Tested here:

* [Platform](https://jenkins.payara.fish/job/JakartaEE-11-TCK/295/)
* [Web Profile](https://jenkins.payara.fish/job/JakartaEE-11-TCK/296/)

Pandrex247 committed:
Fix skipConfig vs. skipServerStartStop, and remove duplicate start

Signed-off-by: Andrew Pielage <pandrex247@hotmail.com>

breakponchito reviewed (approved):
LGTM

breakponchito committed:
adding default value for group and adding profiles to run with correct value for filter

breakponchito commented:
@Pandrex247 I added 3 profiles that can help to configure the Jenkins Job to run with the filtering property, I'm checking on the EngineeringJenkins how to specify the profile when running jakarta data

Pandrex247 committed:
Adjust solution

Utilise existing Web profile, default of Platform, and rename core-data to core

Signed-off-by: Andrew Pielage <pandrex247@hotmail.com>

Pandrex247 commented:
Adjusted the solution, and adjusted the Jenkins job: https://github.com/payara/EngineeringJenkinsjobs/pull/336
